react/DateMonthPicker/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+import { Button, translate, Icon, Bold } from 'cozy-ui/transpiled/react'
+import range from 'lodash/range'
+import { format } from 'date-fns'
+import styles from './styles.styl'
+import cx from 'classnames'
+
+const MonthButton = translate()(({ monthNum, f, onClick, isSelected }) => {
+  const d = new Date(2019, monthNum, 15)
+  const handleClick = () => {
+    onClick(monthNum)
+  }
+  return (
+    <Button
+      theme="secondary"
+      className={cx(
+        styles.DateMonthPicker__MonthButton,
+        isSelected ? styles.ValueSelected : null
+      )}
+      onClick={handleClick}
+      label={f(d, 'MMM')}
+    />
+  )
+})
+
+const useCounter = (initialValue, min, max) => {
+  const [state, setState] = useState(initialValue)
+  const increment = () => setState(Math.min(state + 1, max))
+  const decrement = () => setState(Math.max(state - 1, min))
+  return [state, decrement, increment]
+}
+
+const DateMonthPicker = ({ initialValue, onSelect }) => {
+  const [initialYear, initialMonth] = initialValue
+    .split('-')
+    .map(x => parseInt(x, 10))
```

Comment:
We should add a check and warn the user if the initialValue doesn't match want we want.

Reply:
A PropType check right ?

Reply:
It can be a PropTypes check yes

Reply:
I added a Proptypes check